modules/consumableBidAdapter.js

```diff
@@ -61,7 +61,8 @@ export const spec = {
       source: [{
         'name': 'prebidjs',
         'version': '$prebid.version$'
-      }]
+      }],
+      lang: (navigator && navigator.language) ? navigator.language.split('-')[0] : ''
```

Comment:
this is already on your request object, no need to hit navigator
https://github.com/prebid/Prebid.js/blob/577e8a29e709af3e7fa4464227e7cdc612b24fc9/src/fpd/enrichment.js#L122

Reply:
Updated to grab from the `bidderRequest` object